crates/rust-analyzer/src/reload.rs

```diff
@@ -83,22 +84,22 @@ impl GlobalState {
             if path.extension().unwrap_or_default() != "rs" {
                 return false;
             }
-            if IMPLICIT_TARGET_FILES.iter().any(|it| path.ends_with(it)) {
+            if IMPLICIT_TARGET_FILES.iter().any(|it| path.as_ref().ends_with(it)) {
```

Comment:
Any particular reason to use `as_ref()` here? It papered over the bug above.

Reply:
Was too lazy to convert constant string arrays to relative paths

Reply:
Ah, right, that makes sense.